lib/Doctrine/Migrations/AbstractMigration.php

```diff
@@ -155,6 +166,11 @@ public function getSql() : array
         return $this->plannedSql;
     }
 
+    /**
+     * Write some debug information to the console.
```

Comment:
debug information is written with `debug()`, this is a notice.

Reply:
Fixed